src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -52,6 +52,7 @@ class SonarLintDaemon : ISonarLintDaemon
 
         public event DownloadProgressChangedEventHandler DownloadProgressChanged;
         public event AsyncCompletedEventHandler DownloadCompleted;
+        public event DaemonEventHandler Ready;
```

Comment:
It's recommended to use the generic event handler classes instead of declaring one by yourself 
```
public event EventHandler<EventArgs> Ready;
```